lib/relay/client/dynamic.go

```diff
@@ -176,42 +177,46 @@ type dynamicAnnouncement struct {
 	}
 }
 
-// relayAddressesSortedByLatency adds local latency to the relay, and sorts them
-// by sum latency, and returns the addresses.
-func relayAddressesSortedByLatency(input []string) []string {
-	relays := make(relayList, len(input))
-	for i, relay := range input {
-		if latency, err := osutil.GetLatencyForURL(relay); err == nil {
-			relays[i] = relayWithLatency{relay, int(latency / time.Millisecond)}
-		} else {
-			relays[i] = relayWithLatency{relay, int(time.Hour / time.Millisecond)}
+// relayAddressesOrder checks the latency to each relay, rounds latency down to
+// the closest 30ms, and puts them in buckets of 50ms latency ranges. Then
```

Comment:
That's a weird bucketing algorithm, but I don't think that's what actually happens. The constant `30` doesn't appear in the code.